app/assets/stylesheets/recipe_new.scss

```diff
@@ -0,0 +1,101 @@
+@import "colors";
+
+.recipe-name-input-block textarea {
+  font-size: 16pt;
+  font-family: 'Lobster', cursive;
+  background-color: $white;
+  color: $slate;
+  border-color: darken($light-yellow, 5%);
+  border-radius: 5px;
+}
+
+.recipe-attributes-block {
+  padding: 0 5px;
+  border-radius: 5px;
+  background-color: $dark-orange;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);
+  font-size: 14pt;
+  padding-top: 10px;
+}
+
+.recipe-attributes-block input {
+  background-color: $white;
+  color: $slate;
+  border-color: darken($dark-orange, 5%);
+  border-radius: 5px;
+}
+
+.recipe-attribute-input label {
+  font-size: 10pt;
+}
+
+.recipe-add-categories-block {
+  padding: 0 5px;
```

Comment:
Properties should be ordered background-color, border-radius, box-shadow, font-size, margin, padding, padding-top